finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/service/H2ReqRepFrame.scala

```diff
@@ -32,7 +33,7 @@ case class H2ReqRep(request: Request, response: Try[RepAndFrame]) {
   override def toString: String = s"H2ReqRep($request, $responseOnly, finalFrame=$finalFrame)"
 }
 
-object H2ReqRep {
+object H2ReqRepFrame {
   // type aliases to minimise typing
   type RepAndFrame = (Response, FinalFrame)
   type FinalFrame = Option[Try[Frame]]
```

Comment:
If we have separate `H2ReqRep` and `H2ReqRepFrame` types, is there any reason for this to still be an `Option`?